test/unit/double_free.c

```diff
@@ -70,8 +70,30 @@ TEST_BEGIN(test_large_double_free_no_tcache) {
 }
 TEST_END
 
+TEST_BEGIN(test_double_free_tcache) {
+    test_skip_if(!config_debug);
+
+    test_large_double_free_pre();
+    char *ptr = malloc(SC_NTINY);
```

Comment:
malloc(1) is fine. SC_NTINY is actually the number of tiny size classes.

Reply:
addressed in [4bc176c](https://github.com/jemalloc/jemalloc/pull/2310/commits/4bc176c3002fa751f267c7a420d38a8a2df2e712)